client/src/App.js

```diff
@@ -52,12 +60,14 @@ function App() {
 
   return (
     <MuiThemeProvider theme={theme}>
-      <BrowserRouter>
+      <HashRouter>
```

Comment:
This is fine, I'm just curious as to why you used HashRouter.

Reply:
When there would be a page reload, due to changes saved to file while working, it would go back to "/", HashRouter prevented that, and would take to the current working page.

Reply:
> > Actually I just realized that this is introducing a lot of client errors, so I think it would be best to fix them before merging. I got around 10 different errors. If you need help fixing them I can help.
> 
> Can you share a screenshot of one of the errors?

If you go to the new Polls page and open the console you'll get 12 errors.
Sending a screenshot of 4 errors.
![image](https://user-images.githubusercontent.com/39279007/83980328-df42e580-a8e2-11ea-9ed8-9af2fe2837bd.png)